manifests/enterprise/service/nix.pp

```diff
@@ -16,6 +16,12 @@
       timeout => 0,
       notify  => Exec['enable_splunk'],
     }
+
+    exec { 'disable_splunk_boot_start':
+      command     => "${splunk::enterprise::enterprise_homedir}/bin/splunk disable boot-start --accept-license --answer-yes --no-prompt",
```

Comment:
can you please provide the command as array? see https://www.puppet.com/docs/puppet/8/types/exec.html#exec-attribute-command

Reply:
Sure! I just used the same the string for local consistency.